src/Core/Managed/Shared/TelemetryClientExtensions.cs

```diff
@@ -105,14 +107,60 @@ public static IOperationHolder<T> StartOperation<T>(this TelemetryClient telemet
                 operationTelemetry.Context.Operation.Name = operationTelemetry.Name;
             }
 
+            bool isActivityEnabled = false;
+#if !NET40
+            isActivityEnabled = ActivityProxy.TryRun(() =>
+            {
+                bool operationNameIsSet = false;
+                var activity = new Activity("Internal");
```

Comment:
Is there any significance to the string ```"Internal"```? Maybe it should be more descriptive. e.g. "Application Insights Operation Context". Is there value in keeping it short?